infra/Pos/Network/CLI.hs

```diff
@@ -129,59 +129,64 @@ monitorStaticConfig :: forall m. (
                        , Mockable Catch m
                        )
                     => NetworkConfigOpts
-                    -> T.NodeType     -- ^ Our node type
-                    -> Y.RunKademlia  -- ^ Are we currently running kademlia?
-                    -> Peers NodeId   -- ^ Initial value
+                    -> T.NodeType                  -- ^ Our node type
+                    -> Y.RunKademlia               -- ^ Are we currently running kademlia?
+                    -> Map NodeId T.NodeType       -- ^ Initial directory
+                    -> [(T.NodeType, Alts NodeId)] -- ^ Initial routes
```

Comment:
I thought from https://github.com/serokell/time-warp-nt/pull/90 the new `Peers` type consisted of the routes and the extra node type index. Why not pass a Peers here? Why pass them separately?